src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/DeleteWalletByIdTest.java

```diff
@@ -0,0 +1,68 @@
+package pl.byczazagroda.trackexpensesappbackend.integration;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import pl.byczazagroda.trackexpensesappbackend.BaseIntegrationTestIT;
+import pl.byczazagroda.trackexpensesappbackend.exception.ErrorCode;
+import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransaction;
+import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionType;
+import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
+import pl.byczazagroda.trackexpensesappbackend.repository.FinancialTransactionRepository;
+import pl.byczazagroda.trackexpensesappbackend.repository.WalletRepository;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+
+public class DeleteWalletByIdTest extends BaseIntegrationTestIT {
+
+    @Autowired
+    WalletRepository walletRepository;
+
+    @Autowired
+    FinancialTransactionRepository financialTransactionRepository;
+
+    @BeforeEach
+    void clearDatabase() {
+        walletRepository.deleteAll();
+    }
+
+    @DisplayName("Should delete wallet from database and return status isOk")
```

Comment:
:smile: `.isOK()` **is only the method** of the result of method `status()` from class `MockMvcResultMatchers`:
```java
MockMvcResultMatchers.status().isOk()
```

```java
org.springframework.test.web.servlet

Interface ResultMatcher

	/**
	 * Assert the response status code is {@code HttpStatus.OK} (200).
	 */
	public ResultMatcher isOk() {
		return matcher(HttpStatus.OK);
	}
```

Sooner or later it's verified if the status is `HttpStatus.OK`. The `HttpStatus.OK` code is `200`.


**Link:**
<https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/result/StatusResultMatchers.html#isOk()>
<https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/ResultMatcher.html>
<https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/result/StatusResultMatchers.html>

```java
HttpStatus.OK
```
**Link:**
https://docs.spring.io/spring-framework/docs/3.0.0.M2/javadoc-api/org/springframework/http/HttpStatus.html#OK


So if your `@DisplayName` annotation got value "...... and the return status is XX" - it should get value:
```java
@DisplayName("Should delete wallet from database and return status is `OK`")    // <- my suggestion
or 
@DisplayName("Should delete wallet from database and return status is OK")
or 
@DisplayName("Should delete wallet from database and return status is `200`")
or 
@DisplayName("Should delete wallet from database and return status is 200")
```

Reply:
thanks, I have changed according to your suggestion